src/main/kotlin/io/snyk/plugin/ui/jcef/ToggleDeltaHandler.kt

```diff
@@ -0,0 +1,43 @@
+package io.snyk.plugin.ui.jcef
+
+import com.intellij.ui.jcef.JBCefBrowserBase
+import com.intellij.ui.jcef.JBCefJSQuery
+import io.snyk.plugin.pluginSettings
+import io.snyk.plugin.runInBackground
+import org.cef.browser.CefBrowser
+import org.cef.browser.CefFrame
+import org.cef.handler.CefLoadHandlerAdapter
+import snyk.common.lsp.LanguageServerWrapper
+
+class ToggleDeltaHandler {
+    fun generate(jbCefBrowser: JBCefBrowserBase): CefLoadHandlerAdapter {
+        val toggleDeltaQuery = JBCefJSQuery.create(jbCefBrowser)
+        toggleDeltaQuery.addHandler {deltaEnabled ->
+            if (deltaEnabled.toBoolean()) {
+                pluginSettings().setDeltaEnabled()
```

Comment:
well... why do we need a setter for each? Isn't that a bit too much?

Reply:
I would rather have something like `setDeltaEnabled(enabled:Boolean)` instead of two functions where the only difference is the text assigned.